### ISSUE TITLE
Filter out boosts from people with tons of followers.

### ISSUE THREAD

**Issue by egypturnash** [tootsuite/mastodon#8323]
So one thing I have been really enjoying about Mastodon is the "show boosts" switch. I have found that keeping it off all the time makes my world feel a lot calmer.

I also get this sense that I'm missing things I'd like to see. But I really don't want to turn it back on again now that Twitter Influencers are starting to show up on Mastodon; they're one of the things I was glad to get *away* from.

I'd like to propose a modification to the "show boosts" switch: the ability to show boosts of accounts with less than a user-selectable number of followers.

I envision it looking something like this:

[x] Show boosts from people with less than [1000] followers.

... where [1000] is a dropdown, with suggested values of 1,000 | 5,000 | 10,000 | 100,000 | 1M | 10M | 100M | 1000M. And maybe 'other' which would make a numerical entry box appear so that a user who has decided that 27,984 followers is the precise threshold at which they stop willing to hear about the shenanigans of people they're not following.

Some cursory research suggests that the current largest follow counts on Twitter are between 100M and 110M, which is why I made the list stop at 1000m.

I would also like to suggest that the default setting for new users should be "show boosts from people with less than some number of followers" rather than "show all boosts" as it currently is. The *precise* default might be something an admin could set for her instance in its configuration screens; I feel this might be a valuable tool to gently push your instance towards "talking about their own lives and their local issues" rather than "what that famous person did".

More broadly, I would also sort of love to see this as a filter I could apply to my entire instance if I want to simply opt out of contributing to giving future Mastodon Influencers a bigger megaphone. I kinda want my instance to automatically *silence* anyone it hears about with too many followers. But that's another feature request.

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by egypturnash** [comment]
(sorry about the title editing spam. Getting the precise language is hard, and any suggestions for better phrasing for the actual UI are definitely welcomed!)

**Comment by trwnh** [comment]
I'm not sure this would need to replace the original "show boosts" filter, but it would definitely be useful as an additional option. It might even make more sense to think of it as a separate filter instead of a subset of disabling a certain person's boosts; this feature sounds like it would be more useful as a global filter than having to do it for every single person you are following.

**Comment by egypturnash** [comment]
@trwnh yeah, this is definitely intended as a filter for your entire Mastodon experience rather than your view of one particular person.
